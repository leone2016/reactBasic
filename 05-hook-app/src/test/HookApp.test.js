import React from 'react';
import { shallow } from 'enzyme';
import { HookApp } from '../HookApp'

describe('UNIT TEST HookApp.test.js', () => {
    test('should show the component correctly', () => {
        const wrapper = shallow(<HookApp/>);
        expect(wrapper).toMatchSnapshot();
    })
    
})
