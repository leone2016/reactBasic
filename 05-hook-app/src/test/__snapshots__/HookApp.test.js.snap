// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UNIT TEST HookApp.test.js should show the component correctly 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
